source/clear-linux/tutorials/azure.rst

```diff
@@ -1,125 +1,481 @@
+   clear-linux-os  clear-linux-project  containers        clear-linux-project:clear-linux-os:containers:18620.0.0        18620.0.0
+   clear-linux-os  clear-linux-project  containers        clear-linux-project:clear-linux-os:containers:18860.0.0        18860.0.0
+   clear-linux-os  clear-linux-project  machine-learning  clear-linux-project:clear-linux-os:machine-learning:15780.0.0  15780.0.0
+   clear-linux-os  clear-linux-project  machine-learning  clear-linux-project:clear-linux-os:machine-learning:15960.0.0  15960.0.0
+   clear-linux-os  clear-linux-project  machine-learning  clear-linux-project:clear-linux-os:machine-learning:16050.0.0  16050.0.0
+   clear-linux-os  clear-linux-project  machine-learning  clear-linux-project:clear-linux-os:machine-learning:16150.0.0  16150.0.0
+   clear-linux-os  clear-linux-project  machine-learning  clear-linux-project:clear-linux-os:machine-learning:16500.0.0  16500.0.0
+   clear-linux-os  clear-linux-project  machine-learning  clear-linux-project:clear-linux-os:machine-learning:16810.0.0  16810.0.0
+   clear-linux-os  clear-linux-project  machine-learning  clear-linux-project:clear-linux-os:machine-learning:18080.0.0  18080.0.0
+   clear-linux-os  clear-linux-project  machine-learning  clear-linux-project:clear-linux-os:machine-learning:18620.0.0  18620.0.0
+   clear-linux-os  clear-linux-project  machine-learning  clear-linux-project:clear-linux-os:machine-learning:18860.0.0  18860.0.0
+
+The information shown in the `Urn` column lists the
+`Publisher:Offer:Sku:Version` for each image available and this is the
+information to use to create the |CL| VM. Since we are creating a |CL| basic
+VM, highlight the `clear-linux-project:clear-linux-os:basic:` string and copy
+it to your clipboard. For the version you can use the label `latest` instead
+of referencing a specific version, which is what we will do when we create our
+VM in a moment.
+
+Create a MS Azure resource group
+********************************
+
+With all the information gathered, we need to create a resource group to
+manage multiple resources within MS Azure for our |CL| VM. To learn more about
+resource groups, visit the `Azure Resource Manager overview`_ for an overview
+and detailed description of resources within MS Azure.
+
+To create our new resource group, run the :command:`az` command shown below to
+create a resource group named `ClearResourceGroup` and locate it in the
+`westus` region.
+
+.. code-block:: console
+
+   az group create -n ClearResourceGroup -l westus
+
+When the command has completed, the output from this command is similar to
+the following:
+
+.. code-block:: console
+
+   {
+     "id": "/subscriptions/{unique-id}/resourceGroups/ClearResourceGroup",
+     "location": "westus",
+     "managedBy": null,
+     "name": "ClearResourceGroup",
+     "properties": {
+       "provisioningState": "Succeeded"
+     },
+     "tags": null
+   }
+
```

Comment:
I would have preferred this sequence: 1) Create the resource group 2) Get a list of CL images 3) Create a VM instead of your current sequence of 2), 1), 3).

Reply:
@tpleavitt I understand where you are coming from. The way this is written becomes very confusing. Use numbered steps to indicate how to obtain a particular piece of information and follow them with the steps that use that information. Thus, reader must not keep track of the information or go back to obtain it.